pkg/policy/mapstate.go

```diff
@@ -32,6 +33,14 @@ var (
 	}
 )
 
+const (
+	LabelKeyPolicyDerivedFrom  = "io.cilium.policy.derived-from"
```

Comment:
There's a const already defined with this string. I would suggest to use that `const` or set it in a common package that can be accessed by this one and the users of the already existing `const`

Reply:
I think you are referring to  `github.com/cilium/cilium/pkg/k8s/apis/ciliumio.PolicyLabelDerivedFrom` - however that one is usually of source `k8s` and its string is `io.cilium.k8s.policy.derived-from`, i.e. it contains k8s in the the string.

This one here is for built-in rules, always used with source `reserved`. It has nothing to do with Kubernetes, so I did not want to pull in that.